examples/fhir/oml_message.json

```diff
@@ -931,29 +996,76 @@
             ],
             "city": "DEER CREEK",
             "district": "OTTER TAIL",
-            "state": "MN",
+            "state": "LA",
             "postalCode": "56527-9657",
             "country": "USA"
           }
         ],
-        "multipleBirthBoolean": false
+        "multipleBirthBoolean": false,
+        "contact": [
+          {
+            "relationship": [
+              {
+                "coding": [
+                  {
+                    "system": "http://terminology.hl7.org/CodeSystem/v3-RoleCode",
+                    "code": "MTH",
+                    "display": "mother"
+                  }
+                ]
+              }
+            ],
+            "name": {
+              "text": "SADIE S SMITH",
+              "family": "SMITH",
+              "given": [
+                "SADIE",
+                "S"
+              ]
+            },
+            "telecom": [
+              {
+                "extension": [
+                  {
+                    "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/text",
+                    "valueString": "(763)555-5555"
+                  }
+                ],
+                "system": "phone",
+                "value": "(763) 555 555",
```

Comment:
Missing a digit here

Reply:
I thought it wasn't a bug in RS but instead it is because our original sample ORM/OML messages we were given were missing the last `5` digit.  I've updated those sample files to now have that last `5` digit.  I know my ORM that I sent into RS to make this updated file was also missing the final `5` digit.